services/core-api/app/api/projects/information_requirements_table/models/information_requirements_table.py

```diff
@@ -137,6 +158,14 @@ def update(self, irt_data, import_file=None, document_guid=None, add_to_session=
         if add_to_session:
             self.save()
 
+        if mine_doc and project:
+            if Config.ENVIRONMENT_NAME != 'prod':
```

Comment:
is it possible to squeeze in Simen's feature flag here?

Reply:
This is done.